llvm/tools/opt/optdriver.cpp

```diff
@@ -375,6 +375,7 @@ static bool shouldPinPassToLegacyPM(StringRef Pass) {
       "fix-irreducible",
       "expand-large-fp-convert",
       "callbrprepare",
+      "scalarizer",
```

Comment:
we can use the new pass manager in opt. However, since we are bringing back the legacy pass manager Justin advised me that we should be able to invoke it.  

In fact the original reason this pass was removed was because there were no more scalarizer test cases. If we don't have these tests then someone might come and remove the pass again as untested.